### PR TITLE
Add grafana dashboards for image-controller, image repository provision

### PR DESCRIPTION
**Pull request by rcerven**
[STONEBLD-1775](https://issues.redhat.com//browse/STONEBLD-1775)

### CODE REVIEW

File: components/monitoring/grafana/base/dashboards/image-controller/OWNERS [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# See the OWNERS docs: https://go.k8s.io/owners
+# Should be kept in sync with components/monitoring/grafana/base/dashboards/image-controller/OWNERS
+
+approvers:
```
Comment (mkosiarc) [inline components/monitoring/grafana/base/dashboards/image-controller/OWNERS:4]: What about Sushanta and Adam?

File: components/monitoring/grafana/base/dashboards/image-controller/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+#TODO
```
Comment (mkosiarc) [inline components/monitoring/grafana/base/dashboards/image-controller/kustomization.yaml:4]: Is this waiting for another PR?

Comment (rcerven) [reply components/monitoring/grafana/base/dashboards/image-controller/kustomization.yaml:4]: yes for https://github.com/redhat-appstudio/image-controller/pull/85

Comment (rcerven) [reply components/monitoring/grafana/base/dashboards/image-controller/kustomization.yaml]: updated

File: components/monitoring/grafana/base/dashboards/image-controller/OWNERS [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+- susdas
+
+reviewers:
+- acmiel
+- mmorhun
+- tkdchen
+- rcerven
+- mkosiarc
+- stuartwdouglas
+- susdas
```
Comment (tisutisu) [inline components/monitoring/grafana/base/dashboards/image-controller/OWNERS:20]: Its my `redhat user id` .. do we need to specify `github user id`?

Comment (rcerven) [reply components/monitoring/grafana/base/dashboards/image-controller/OWNERS:20]: I am not sure

Comment (hugares) [reply components/monitoring/grafana/base/dashboards/image-controller/OWNERS:20]: github

Comment (rcerven) [reply components/monitoring/grafana/base/dashboards/image-controller/OWNERS]: updated with github name

**Pull request merged**